rerun_py/rerun_sdk/rerun/archetypes/asset_video_ext.py

```diff
@@ -59,3 +63,19 @@ def __init__(
             return
 
         self.__attrs_clear__()
+
+    def read_frame_timestamps_ns(self: Any) -> npt.NDArray[np.int64]:
+        """
+        Determines the presentation timestamps of all frames inside the video.
+
+        Throws a runtime exception if the video cannot be read.
+        """
+        if self.blob is not None:
+            video_buffer = self.blob.as_arrow_array()
+        else:
+            raise RuntimeError("Asset video has no video buffer")
+
+        if self.media_type is not None:
+            media_type = self.media_type.as_arrow_array().storage[0].as_py()
```

Comment:
I think `to_numpy()` could be more efficient here (if it works).  `as_py()` will convert to a python list which I think carries some more overhead, though given the data is known to be uniform, it's possible the interpreter does something smart here.

Reply:
will check!

Reply:
`AttributeError: 'pyarrow.lib.StringScalar' object has no attribute 'to_numpy'`
so, no